### ISSUE TITLE
n --download outputs info to STDERR with no option to silence

### ISSUE THREAD

**Issue by gauthierm** [tj/n#653]
# Bug Report

## Summary

The `n --download [version]` command outputs a summary to STDERR. In scripts, this produces extra noise that is difficult to turn off. The `--quiet` flag does not affect this output. Since STDERR is used, redirecting output could suppress actual error information

## Steps to Reproduce

```sh
n --download --quiet [version]
```

## Expected Behaviour

no output unless there is an actual error downloading the version

## Actual Behaviour

STDERR outputs
```
       found : /Users/myusername/my-project/package.json
        read : ^14.15.1
      target : 14
```

## Other Information

If the `--quiet` flag can't be used, the non-error output should at least be output on STDOUT.

## Configuration Details

```bash
$ n --version
7.0.1

$ command -v node
/Users/myusername/.n/bin/node

$ node -p process.platform
darwin
```

**Comment by shadowspawn** [comment]
1) Yes, historically the `--quiet` flag just disables the download progress from `curl`. There is not currently a way of turning off the log messages.

2) I am seeing the log messages on STDOUT and not STDERR?

```
% n lts
   installed : v14.15.5 (with npm 6.14.11)
% n lts > /dev/null
% n 99 > /dev/null

  Error: no version found for '99'

```

**Comment by jaymefSO** [comment]
```
The problem is with other n commands such as n --download or n exec

ex:

```
$ n --quiet --download engine >/dev/null
       found : package.json
        read : ^14.15.1
      target : 14
```

Why should this have standard output?

similiarily:

```
$ n --quiet exec engine >/dev/null
       found : package.json
        read : ^14.15.1
      target : 14
```

**Comment by gauthierm** [comment]
The code uses:

```
log() {
  printf "  ${SGR_CYAN}%10s${SGR_RESET} : ${SGR_FAINT}%s${SGR_RESET}\n" "$1" "$2"
}
trace_log() {
  >&2 log "$1" "$2"
}
```

It looks like a flag could be added to that fairly easily.

**Comment by shadowspawn** [comment]
Oh, I see the difference. The log output from the (newer) `auto` and `engine` support is indeed going to `STDERR` for implementation reasons. I'll take a look at that. (Uses `trace_log`.)

**Comment by shadowspawn** [comment]
I'm working on changing the `auto` and `engine` logging to use `STDOUT` like the rest of the logs.

Being able to suppress the logging with an option, say by using `--quiet`, is also possible. But I suggest open another issue if that is still of interest after logging is consistently going to `STDOUT`.

(The `auto` and `engine` logging is relatively verbose, so can make sense of what happened when it does not work as expected.)

```
$ n --download engine >/dev/null
$ n --download auto >/dev/null
```

**Comment by shadowspawn** [comment]
Released in 7.0.2
